packages/colorpickers/src/styled/ColorPicker/StyledSliders.ts

```diff
@@ -24,19 +24,24 @@ export const StyledSliders = styled.div.attrs({
   margin-${props => (props.theme.rtl ? 'right' : 'left')}: ${props => props.theme.space.base * 2}px;
   width: 100%;
 
-  & > * {
-    position: absolute;
-    width: 100%;
-    height: ${props => getTrackMargin(props) * 2 + getTrackHeight(props)}px;
-  }
-
-  & > :first-child {
-    top: -${getTrackMargin}px;
-  }
-
-  & > :last-child {
-    bottom: -${getTrackMargin}px;
-  }
+  ${props =>
+    !props.$isOpaque &&
+    css`
```

Comment:
Is `css` necessary here?

Reply:
This provides proper placement for the sliders when `isOpaque` is `false`.

Reply:
yep, I get that. More wondering whether this can be a simple template literal rather than invoking the `css` function?

Reply:
I tried the template literal first but it didn't parse correctly. I'll give it another shot.

Reply:
...as I understand it, `styled` already handles this, making `css` redundant...

> during flattening, styled-components ignores interpolations that evaluate to undefined, null, false, or an empty string ("")

https://styled-components.com/docs/advanced#tagged-template-literals